src/types/SortType.ts

```diff
@@ -0,0 +1,5 @@
+export enum SortType {
+  ALPHABET,
+  LENGTH,
+  Default,
```

Comment:
The `Default` enum value should be renamed to `DEFAULT` to follow the convention of using uppercase for enum values, indicating that they are constants.